### ISSUE TITLE
Redundant blank lines under the title of embedded note

### ISSUE THREAD

**Issue by u2003** [kepano/obsidian-minimal#116]
I don't know what's wrong with my notes, but when I try to embed another note into the current one, I find redundant blank lines under the embedded title. (As you can see in the picture) I ask some of my friends who use Minimal as well, and they tell me that they don't encounter the same problem. I'm now very confused, it would be so great if you could please help me with this problem~❤️
<img width="805" alt="image" src="https://user-images.githubusercontent.com/84957135/149103905-9a243d34-6470-460b-a02a-9028671b9f54.png">

**Comment by kepano** [comment]
Can you please provide the debug info?

It can be accessed in Obsidian with Ctrl/Cmd-P, enter "Show debug info", and click "Copy to clipboard".

**Comment by u2003** [comment]
Thank you for your early reply! I did try but I didn't find such command, could you please give me more detailed instructions?

**Comment by u2003** [comment]
omg I finally find the command and you can see it in the following. Please forgive me for being so late🥲 I just didn't find this command via "show debug info" because I didn't know it had been translated into my mother tongue, as you can see my Obsidian is currently bilingual.

Thank you for your support once again~🥰

SYSTEM INFO:
	Obsidian version: v0.13.19
	Installer version: v0.13.19
	Login status: logged in
	Catalyst license: none
	Insider build toggle: off
	Live preview: on
	Legacy editor: off
	Base theme: light
	Community theme: Minimal
	Snippets enabled: 3
	Safe mode: off
	Plugins installed: 139
	Plugins enabled: 83
		1: Homepage
		2: Gallery
		3: Breadcrumbs
		4: Typewriter Scroll
		5: Easy Typing
		6: Outliner
		7: Auto pair chinese symbol
		8: MarginNote Companion
		9: Recent Files
		10: Fullscreen mode plugin
		11: Zoom
		12: Word Splitting for Simplified Chinese in Edit Mode
		13: LanguageTool Integration
		14: Calendar
		15: Advanced Tables
		16: DEVONlink
		17: Quick Switcher++
		18: Table of Contents
		19: Convert url to preview (iframe)
		20: Citations
		21: Hotkey Helper
		22: Media Extended
		23: Note Refactor
		24: Paste URL into selection
		25: Tag Wrangler
		26: Better footnote
		27: Footlinks
		28: Natural Language Dates
		29: Footnote Shortcut
		30: Icons
		31: Tidy Footnotes
		32: Style Settings
		33: QuickAdd
		34: Hider
		35: Activity History
		36: Command Alias
		37: Mind Map
		38: Copy Block Link
		39: Banners
		40: Dataview
		41: Icon Folder
		42: Sliding Panes (Andy's Mode)
		43: MetaEdit
		44: Hotkeys for templates
		45: Media Extended BiliBili Plugin
		46: Smart Random Note
		47: Customizable Sidebar
		48: Kanban
		49: Cycle through Panes
		50: Open with
		51: Admonition
		52: Number Headings
		53: Table Extended
		54: Flashcards
		55: Force note view mode
		56: Graph Analysis
		57: Journey
		58: Customizable Menu
		59: Emoji Toolbar
		60: Markdown prettifier
		61: Get Info
		62: Smart Typography
		63: Advanced URI
		64: Excalidraw
		65: Custom window title
		66: Auto Link Title
		67: Customizable Page Header
		68: Juggl
		69: Global Hotkeys
		70: Charts View
		71: Contextual Typography
		72: Sortable
		73: System Dark Mode
		74: Plugin Changelogs
		75: Pandoc Plugin
		76: Obsidian42 - BRAT
		77: Frontmatter Tag Suggest
		78: Obsidian Charts
		79: Spaced Repetition
		80: Vantage - Advanced search builder
		81: Obsidian Memos
		82: Periodic Notes
		83: Image auto upload Plugin

RECOMMENDATIONS:
	Custom theme: for cosmetic issues, please try to make the issue happen in the help vault or disable community theme and snippets.
	Community plugins: for bugs, please try to make the issue happen in the help vault or disable community plugins.

**Comment by kepano** [comment]
You have 83 plugins and 3 snippets enabled 😳

Please try in a new vault with only Minimal Theme to see if this comes up. Or turn off all plugins/snippets and enable them one by one to see which one causes the issue.

**Comment by u2003** [comment]
I've tried now, and it's ok in the new vault, maybe it's contradicted by another plugin. I'll try them one by one to see which it is. Thank you so much~

**Comment by u2003** [comment]
oh I've got it after testing all the plugins one by one. It's because I once mistakenly enabled the plugin Style Settings and Minimal Theme Settings simultaneously, and after I disabled the latter, this problem of redundant blank lines came up.

**Comment by u2003** [comment]
I'm sorry to keep bothering you, but the problem is not solved yet. After I tested a few more times, I think it's not because of the contradiction between Style Settings and Minimal Style Settings, but the problem of Focus Mode. Actually, I downloaded the Minimal Style Settings long time ago, and after I found it's almost the same as Style settings, I quickly disabled it. When I test in a new vault, the issue does come up once, but later I realize it's because Focus Mode is enabled in one of them. Now I find even I only have Style Settings enabled (or Minimal Style Settings, just the same), as long as I toggle Focus Mode, this problem will happen. When I disable it, redundant blank lines will disappear automatically.
